Microsoft.Toolkit.Uwp.SampleApp/Shell.xaml.cs

```diff
@@ -427,7 +454,7 @@ private void NavigationFrameOnNavigated(object sender, NavigationEventArgs navig
                 this._lastRenderedProperties = true;
 
                 // Called to load the sample initially as we don't get an Item Pivot Selection Changed with Sample Loaded yet.
-                UpdateXamlRenderAsync(_currentSample.BindedXamlCode);
+                var t = UpdateXamlRenderAsync(_currentSample.BindedXamlCode);
```

Comment:
vat t is unused

Reply:
quick workaround to remove the VS warning about calling an async method without await.